### PR TITLE
[COMMON BUG] register bookmarks celery task

### PR DESCRIPTION
**Pull request by Golub-Sergey**
**Description:** Register bookmarks celery tasks for update cms xblock cache.

**Youtrack:** [ticket](https://youtrack.raccoongang.com/issue/CB-112)

**Testing instructions:**

1. Open CMS and SignIN
2. Create course and add Section/Subsection/Unit
3. Open LMS and login
4. Enroll on created course and add this unit in bookmarks on the "course" page LMS
5. Change names for Section/Subsection/Unit on Section1/Subsection1/Unit1
6. Check "Bookmarks" page

### CODE REVIEW

File: openedx/core/djangoapps/bookmarks/apps.py [hunk 1]
```diff
@@ -32,3 +32,7 @@ class BookmarksConfig(AppConfig):
     def ready(self):
         # Register the signals handled by bookmarks.
         from . import signals
+        # Register the tasks handled by signals.
+        from . import tasks
+        from cms import CELERY_APP
```
Comment (flying-pi) [inline openedx/core/djangoapps/bookmarks/apps.py]: Could you wrap it with try-except block to handling import error?

Comment (flying-pi) [inline openedx/core/djangoapps/bookmarks/apps.py]: Could you wrap it with try-except block to handling import error?

Comment (Golub-Sergey) [reply openedx/core/djangoapps/bookmarks/apps.py]: import of module and setting works correctly

Comment (flying-pi) [reply openedx/core/djangoapps/bookmarks/apps.py]: Ok, thakyou

**Comment by cmltaWt0** [comment]
Do you plan to squash commits?

**Comment by idegtiarov** [comment]
@cmltaWt0 Yes, I do plan squash commits.

**Pull request merged**